java/transforms-by-suitability/src/main/java/com/esri/arcgisruntime/sample/transformsbysuitability/DatumTransformationAdapter.java

```diff
@@ -0,0 +1,96 @@
+package com.esri.arcgisruntime.sample.transformsbysuitability;
+
+import java.util.ArrayList;
+
+import android.content.Context;
+import android.graphics.Typeface;
+import android.text.TextUtils;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.TextView;
+
+import com.esri.arcgisruntime.geometry.DatumTransformation;
+import com.esri.arcgisruntime.geometry.GeographicTransformation;
+import com.esri.arcgisruntime.geometry.GeographicTransformationStep;
+
+/**
+ * Adapter that takes an ArrayList of DatumTransformations and shows the name of each transformation, and details
+ * of missing grid files, if any.
+ */
+public class DatumTransformationAdapter extends ArrayAdapter<DatumTransformation> {
```

Comment:
No, but it seems we generally use this style throughout our samples, I was assuming so that classes might be easily pulled in to other codebases by users. What do you think - stick to this?

Reply:
Ahh yeah, thats a sensible use case. *thumbs up*